frontend/style.css

```diff
@@ -31,16 +31,16 @@ body {
     grid-template-columns: 350px;
     grid-template-rows: 45px 45px 45px 45px 45px 45px 45px;
     align-items: end;
-    /* justify-items: start; */
```

Comment:
Remove the comments. If you want to keep it as reference, then either save it somewhere else or using stage keep it locally.